Cargo.toml

```diff
@@ -41,9 +47,9 @@ bencher = "0.1.5"
 getrandom = { version = "0.2.14", features = ["js"] }
 walkdir = "2.5.0"
 time = { version = "0.3.34", features = ["formatting", "macros"] }
-
+anyhow = "1"
```

Comment:
Please make this a dev dependency, since it's unused outside of the examples.

Reply:
Already is?